doc/neptune-metadata.rst

```diff
@@ -0,0 +1,75 @@
+===========================
+Neptune Metadata Definition
+===========================
+
+This file specifies what fields are in the ``.neptune.yml`` file, what they mean
+and by which tools they are used. The file itself is expected to be in a
+repositories root directory.
+
+.. contents::
+
+library
+-------
+
+Type
+  boolean
+Meaning
+  If set to true, the repository is marked as a library used by other repositories
+Tools
+  - neptune-overview
+  - neptune-autopr
+Example
+  .. code:: yml
+
+    library: true
+
+d2ready
+-------
+
+Type
+  boolean
+Meaning
+  If set to true, the repository is marked as ready for D2 compilation
+Tools:
+  - Makd
```

Comment:
Not really, `beaver` handles this (and other CIs). Or should, I'm not really sure it does. (we had a `.D2-ready` file or something like that to mark repos before too, I'm not sure we are using the `neptune.yml` file for that already).

Yeah, confirmed, beaver is not using it yet: https://github.com/sociomantic-tsunami/beaver/blob/v0.x.x/lib/dlang.sh#L39

Reply:
So who is actually using this then? neptune-overview?